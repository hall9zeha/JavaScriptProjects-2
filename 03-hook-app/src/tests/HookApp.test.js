import renderer from "react-test-renderer";
import { HookApp } from "../HookApp";

describe('Tests in <HookApp/> component', () => { 

    test('should show HookApp successful', () => { 
        const wrapper = renderer.create(<HookApp/>).toJSON();
        expect(wrapper).toMatchSnapshot();
     })

 });