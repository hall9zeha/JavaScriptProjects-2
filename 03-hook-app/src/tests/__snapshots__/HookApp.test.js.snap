// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HookApp/> component should show HookApp successful 1`] = `
<div>
  HookApp
</div>
`;
